src/utils/inMemoryDB.ts

```diff
@@ -9,18 +9,19 @@ import { logger } from '../logger.js'
  * More portable than requiring user to set up Mongo in a background Docker process.
  * Need a replset to faciliate transactions.
  */
-const mongod = await MongoMemoryReplSet.create({
-  // Stream listener listens on DB denoted by 'MONGO_DBNAME' env var.
-  replSet: { count: 1, storageEngine: 'wiredTiger', dbName: checkVar('MONGO_DBNAME') }
-})
+let mongod: MongoMemoryReplSet
```

Comment:
Moving this into `connect` fixes a long-standing bug where jest tests don't exit within a second of finishing due to `openHandles`. When we import these tests, we were creating Mongo connections that were not necessarily closed later on. Now, we only create them on connection, so there is less inadvertent creation.